src/get-own-property-symbols.js

```diff
@@ -125,86 +124,97 @@
     }
   ;
 
-  descriptor.value = $defineProperty;
-  defineProperty(Object, DP, descriptor);
+  defineProperty(Object, DP, {
+    value: $defineProperty
```

Comment:
This seems like it will break code by not getting the original descriptor, and thus setting configurable/enumerable/writable to false.

Reply:
Ah good call, yes makes sense! Will take a look.